fh-statsd/config/ose-placeholders.js

```diff
@@ -6,7 +6,7 @@ var placeholders = {
     "streams": [{
       "type": "stream",
       "src": true,
-      "level": "$(FH_LOG_LEVEL)",
+      "level": "{{env.FH_LOG_LEVEL}}",
```

Comment:
@grdryn Assuming this change in syntax is OK.

Reply:
@mikenairn Yeah, this was needed because of the update to fh-config. Without this, it fails to start in RHMAP4, with this change it starts. 👍